addons/website/static/src/components/configurator/configurator.js

```diff
@@ -110,6 +113,34 @@ class DescriptionScreen extends Component {
         }
     }
 
+    clearIndustrySelection() {
+        this.industrySelection.el.value = '';
+        this.industrySelection.el.parentNode.dataset.value = '';
+        this.dispatch('selectIndustry', undefined, undefined);
+    }
+
+    setSelectedIndustry(label) {
+        this.industrySelection.el.parentNode.dataset.value = label;
+        const id = this.labelToId[label];
+        this.dispatch('selectIndustry', label, id);
+        this.checkDescriptionCompletion();
+    }
+
+    customizeMenu(_) {
+        const items = $(".ui-menu a");
+        if (!this.autocompleteHasResults) {
+            items.addClass("no-result");
+        } else {
+            items.removeClass("no-result");
+        }
+    }
```

Comment:
Choose:
- I want to use jquery:
    - Name your variable by starting it with `$`
    - Use `toggleClass` instead of "if..add else..remove"
- I can get rid of jquery (good choice):
    - Suffix your varaible by El or Els (for "elements")
    - Use classList.toggle instead of "if..add else..remove"

Also, can't you avoid searching in the whole DOM? Don't you have a reference to that menu your are searching in?